Formula/jobber.rb

```diff
@@ -0,0 +1,41 @@
+class Jobber < Formula
+  desc "Alternative to cron, with better status-reporting and error-handling"
+  homepage "https://dshearer.github.io/jobber/"
+  url "https://github.com/dshearer/jobber/archive/v1.4.1.tar.gz"
+  sha256 "891b1f86f65690b36cfd7f0c3c78f9d81fdc976e85a8055ac4da0daf59adf066"
+  head "https://github.com/dshearer/jobber.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "install", "prefix=#{prefix}", "bindir=#{bin}", "libexecdir=#{libexec}", "sysconfdir=#{etc}"
+  end
+
+  plist_options :startup => true
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>Program</key>
+          <string>#{libexec}/jobbermaster</string>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>KeepAlive</key>
+          <dict>
+            <key>Crashed</key>
+            <true/>
+          </dict>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    assert_equal "1.4.1", shell_output("#{bin}/jobber -v").chomp
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
Same if the software requires a virtual machine, docker instance, etc. to be running.